src/components/v2/appDetails/k8Resource/nodeType/PodPopup.tsx

```diff
@@ -21,7 +21,7 @@ import { PodPopupProps } from './types'
 import { ReactComponent as ICDeleteInteractive } from '../../../../../assets/icons/ic-delete-interactive.svg'
 import { NodeType } from '../../appDetails.type'
 
-const OpenSecurityModalButton = importComponentFromFELibrary('OpenSecurityModalButton')
+const OpenSecurityModalButton = importComponentFromFELibrary('OpenSecurityModalButton', null, 'function')
```

Comment:
Why are we expecting it as a function?

Reply:
have added export as default, is being exported as function